lifecycle_test.go

```diff
@@ -309,16 +311,24 @@ func remoteDeleteReleaseBundle(t *testing.T, lcManager *lifecycle.LifecycleServi
 }
 */
 
-func uploadBuild(t *testing.T, specFileName, buildName, buildNumber string, uploadAsDependencies bool) {
+func uploadBuildWithArtifacts(t *testing.T, specFileName, buildName, buildNumber string) {
 	specFile, err := tests.CreateSpec(specFileName)
 	assert.NoError(t, err)
 
-	if uploadAsDependencies {
-		runRt(t, "upload", "--spec="+specFile)
-		assert.NoError(t, lcCli.WithoutCredentials().Exec("rt", "bad", buildName, buildNumber, tests.RtDevRepo+"/c*.in", "--from-rt"))
-	} else {
-		runRt(t, "upload", "--spec="+specFile, "--build-name="+buildName, "--build-number="+buildNumber)
-	}
+	runRt(t, "upload", "--spec="+specFile, "--build-name="+buildName, "--build-number="+buildNumber)
+	runRt(t, "build-publish", buildName, buildNumber)
+}
+
+func uploadBuildWithDeps(t *testing.T, buildName, buildNumber string) {
+	err := fileutils.CreateDirIfNotExist(tests.Out)
+	assert.NoError(t, err)
+
+	randFile, err := io.CreateRandFile(filepath.Join(tests.Out, "dep-file"), 1000)
```

Comment:
Will you be able to create a PR for the gofrog project, adding a description comment for the `CreateRandFile` function? It'll make the code easier to read, as the name of the function isn't completely explanatory.